src/Uno.Templates/content/unoapp/global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "msbuild-sdks": {
+    "Uno.Sdk": "$UnoWinUIVersion$",
```

Comment:
```suggestion
    "Uno.Sdk": "$UnoVersion$",
```
Should this be `$UnoVersion$` ?

Reply:
No that isn't what the Template uses to update the version